### PR TITLE
clusters/*: adding golang 1.20 image

### PR DESCRIPTION
**Pull request by joelanford**

### CODE REVIEW

File: ci-operator/jobs/infra-periodics-origin-release-images.yaml [hunk 1]
```diff
@@ -155,6 +155,32 @@ periodics:
...
+  spec:
+    containers:
+    - args:
+      - --namespace=ci
+      - start-build
+      - origin-release-golang-1.20
+      - --wait=true
+      command:
+      - oc
+      image: registry.ci.openshift.org/ocp/4.8:cli
```
Comment (bparees) [inline ci-operator/jobs/infra-periodics-origin-release-images.yaml:177]: 4.8?

Comment (joelanford) [reply ci-operator/jobs/infra-periodics-origin-release-images.yaml:177]: Yeah, I noticed that too. Cargo-culted from previous golang-* image additions. I also had a similar question about the `el7` repo being used...

File: clusters/build-clusters/01_cluster/ci/_origin-release-build/golang-1.20/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ENV VERSION=1.20.5 \
+    GOCACHE=/go/.cache \
+    GOARM=5 \
+    GOPATH=/go \
+    GOROOT=/usr/local/go \
+    LOGNAME=deadbeef
+
+ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
+
+COPY cbs-paas7-openshift-multiarch-el7-build.repo /etc/yum.repos.d/
```
Comment (joelanford) [inline clusters/build-clusters/01_cluster/ci/_origin-release-build/golang-1.20/Dockerfile:18]: `el7` is fairly old now, right? Should we use a more recent repo here?

**Comment by joelanford** [comment]
I actually don't think I need this anymore. I think I can use `rhel-8-release-golang-1.20-openshift-4.14` instead.

Not sure what the differences are, and not sure if I should close this or leave it open for others...

**Comment by danilo-gemoli** [comment]
> I actually don't think I need this anymore. I think I can use rhel-8-release-golang-1.20-openshift-4.14 instead. 
Not sure what the differences are, and not sure if I should close this or leave it open for others...

PR got stale, I think we can close it.
We can always come back to it whenever we need.
/close

**Pull request closed without merging**